vpr/src/base/vpr_constraints_serializer.h

```diff
@@ -0,0 +1,232 @@
+#ifndef VPR_CONSTRAINTS_SERIALIZER_H_
+#define VPR_CONSTRAINTS_SERIALIZER_H_
+
+#include "region.h"
+#include "vpr_constraints.h"
+#include "partition.h"
+#include "partition_region.h"
+#include "echo_files.h"
+#include "constraints_load.h"
+#include "vtr_log.h"
+#include "globals.h" //for the g_vpr_ctx
+
```

Comment:
Have a block comment at the top explaining which functions go with read functionality, and which go with write functionality, etc.

Reply:
can point to the rr_graph tutorial for more info